src/test/ui/proc-macro/nested-macros/capture-macro-rules-invoke.rs

```diff
@@ -0,0 +1,22 @@
+// aux-build:recollect.rs
+// check-pass
+
+extern crate recollect;
+use recollect::*;
+
+macro_rules! use_expr {
+    ($expr:expr) => {
+        recollect!($expr)
```

Comment:
The behavior of this test is actually affected by this PR, even without https://github.com/rust-lang/rust/pull/73084/ (it doesn't compile on the latest nightly). I'm not sure if this is worth a Crater run.